docs/file-reference.md

```diff
@@ -314,6 +317,18 @@ secrets:
 | array of [secret](#secret) | no           | [secret](#secret) object |
 
 
+
+## imageStreams
+
+```yaml
+imageStreams:
+- <imageStreamObject>
+```
+
+| Type                         | Required | Description |
+|----------------------------------|--------------|------|
+| array of [imageStream Object](#imageStreamObject) | no           | [imageStream Object](#imageStreamObject) object |
```

Comment:
link reference has to be lowercase `[imageStream Object](#imagestreamobject)`

Reply:
fixed